### DISCUSSION TITLE
Negative Rolling Disc Penalty force

### DISCUSSION THREAD

**Discussion by ManuelZ** [jgerstmayr/EXUDYN#72] [Q&A on usage of Exudyn]
I have this simulation, where the part of interest for this question, is one wheel:

https://github.com/jgerstmayr/EXUDYN/assets/115771/8e0c4f00-456f-4e24-b11a-496b115b7915

I'm getting this contact force on the left wheel:
![Contact force](https://github.com/jgerstmayr/EXUDYN/assets/115771/cd652774-4310-47de-b142-a142c17eaaba)

But I don't understand why the longitudinal force is negative. It should be positive in the J1 system of reference of the RollingDiscPenaly object.

From the ForceLocal J1^f = [f_t,x, f_t,y, f_n] I'm plotting f_t,y and f_n. In this case, f_t,y is the same as W_2, and in this reference frame, I'm getting a negative value, but I expect it to be positive.

![image](https://github.com/jgerstmayr/EXUDYN/assets/115771/e1ea312e-2983-448a-b5cf-d3ee67e25896)

Does this make sense? Am I making a simple mistake somewhere?

**Comment by jgerstmayr** [comment]
I cannot reproduce exactly, what you are modeling. There are some details missing, such as if the wheel rotation is free (you should attach some graphics to visualize the rotation). 
The longitudinal force can be negative, if it acts against w_2 on the wheel. Furthermore, you should display the axis of the wheel. If the axis direction is switched, it is likely that w_2 points in the opposite direction ...
